kolibri/plugins/coach/assets/src/views/reports/ReportsLearnerReportLessonPage.vue

```diff
@@ -31,12 +30,16 @@
             <LessonActive :active="lesson.active" />
           </template>
         </HeaderTableRow>
-        <!-- TODO COACH
         <HeaderTableRow>
-          <template slot="key">{{ coachCommon$tr('descriptionLabel') }}</template>
-          <template slot="value">Ipsum lorem</template>
+          <template slot="key">
+            {{ coachCommon$tr('descriptionLabel') }}
+          </template>
+          <template slot="value">
+            <span dir="auto">
+              {{ lesson.description || coachCommon$tr('descriptionMissingLabel') }}
+            </span>
+          </template>
         </HeaderTableRow>
-         -->
```

Comment:
was this completely broken before?

Reply:
There are a lot of commented out sections like this in different report pages.